contrib/pom.xml

```diff
@@ -228,7 +228,7 @@
     <dependency>
       <groupId>org.apache.kafka</groupId>
       <artifactId>kafka_2.10</artifactId>
-      <version>0.8.1.1</version>
```

Comment:
The support for offset storage in topics is only available in 0.8.2. Rest should work by 0.8.2 client talking to 0.8.1 server.